default_settings.json

```diff
@@ -0,0 +1,5254 @@
+	//  - onFirstSessionStart: Show debug in status bar only after debug was started for the first time
+	"debug.showInStatusBar": "onFirstSessionStart",
+
+	// Controls whether the debug sub-sessions are shown in the debug tool bar. When this setting is false the stop command on a sub-session will also stop the parent session.
+	"debug.showSubSessionsInToolBar": false,
+
+	// Before starting a new debug session in an integrated or external terminal, clear the terminal.
+	"debug.terminal.clearBeforeReusing": false,
+
+	// Controls the location of the debug toolbar. Either `floating` in all views, `docked` in the debug view, or `hidden`.
+	"debug.toolBarLocation": "floating",
+
+	// Global debug launch configuration. Should be used as an alternative to 'launch.json' that is shared across workspaces.
+	"launch": {
+		"configurations": [],
+		"compounds": []
+	},
+
+	// Enable/disable autoclosing of HTML tags.
+	"html.autoClosingTags": true,
+
+	// Enable/disable auto creation of quotes for HTML attribute assignment. The type of quotes can be configured by `html.completion.attributeDefaultValue`.
+	"html.autoCreateQuotes": true,
+
+	// Controls the default value for attributes when completion is accepted.
+	//  - doublequotes: Attribute value is set to "".
+	//  - singlequotes: Attribute value is set to ''.
+	//  - empty: Attribute value is not set.
+	"html.completion.attributeDefaultValue": "doublequotes",
+
+	// A list of relative file paths pointing to JSON files following the [custom data format](https://github.com/microsoft/vscode-html-languageservice/blob/master/docs/customData.md).
+	// 
+	// VS Code loads custom data on startup to enhance its HTML support for the custom HTML tags, attributes and attribute values you specify in the JSON files.
+	// 
+	// The file paths are relative to workspace and only workspace folder settings are considered.
+	"html.customData": [],
+
+	// List of tags, comma separated, where the content shouldn't be reformatted. `null` defaults to the `pre` tag.
+	"html.format.contentUnformatted": "pre,code,textarea",
+
+	// Enable/disable default HTML formatter.
+	"html.format.enable": true,
+
+	// End with a newline.
+	"html.format.endWithNewline": false,
+
+	// List of tags, comma separated, that should have an extra newline before them. `null` defaults to `"head, body, /html"`.
+	"html.format.extraLiners": "head, body, /html",
+
+	// Format and indent `{{#foo}}` and `{{/foo}}`.
+	"html.format.indentHandlebars": false,
+
+	// Indent `<head>` and `<body>` sections.
+	"html.format.indentInnerHtml": false,
+
+	// Maximum number of line breaks to be preserved in one chunk. Use `null` for unlimited.
+	"html.format.maxPreserveNewLines": null,
+
+	// Controls whether existing line breaks before elements should be preserved. Only works before elements, not inside tags or for text.
+	"html.format.preserveNewLines": true,
+
+	// Honor django, erb, handlebars and php templating language tags.
+	"html.format.templating": false,
```

Comment:
?! true

Reply:
@raghubetina I believe this just handles syntax highlighting. Without `htmlbeautifier` or the extension installed I'm prompted to choose a formatter—

<img width="739" alt="Screen Shot 2022-04-25 at 4 06 02 PM" src="https://user-images.githubusercontent.com/17581658/165174987-f823c8d2-244b-40f5-a406-8c9be89f416b.png">